packages/flutter_tools/flutter_analysis_options

```diff
@@ -28,25 +30,25 @@ analyzer:
 
```

Comment:
similarly, remove the parenthetical on "strong_mode_invalid_field_override"'s comment